source/platforms/databricks.md

```diff
@@ -20,42 +43,20 @@ In order to launch a GPU node uncheck **Use Photon Acceleration**.
 
 ![Screenshot of Use Photon Acceleration unchecked](../images/databricks-deselect-photon.png)
```

Comment:
It looks like we missed a step here where we tell folks which Databricks Runtime to choose. Can you add an extra paragraph with this information?